python/GafferSceneUI/OptionQueryUI.py

```diff
@@ -384,12 +361,11 @@ def __addFromGlobalsMenuDefinition( self ) :
 
 		result = IECore.MenuDefinition()
 
-		node = _optionQueryNode( self )
```

Comment:
Same question here about whether or not we should use `findDestination()`.

Reply:
As above...